core/src/main/resources/org/apache/spark/ui/static/sorttable.js

```diff
@@ -81,18 +81,18 @@ sorttable = {
       if (!headrow[i].className.match(/\bsorttable_nosort\b/)) { // skip this col
         mtch = headrow[i].className.match(/\bsorttable_([a-z0-9]+)\b/);
         if (mtch) { override = mtch[1]; }
-        if (mtch && typeof sorttable["sort_"+override] == 'function') {
-          headrow[i].sorttable_sortfunction = sorttable["sort_"+override];
-        } else {
-          headrow[i].sorttable_sortfunction = sorttable.guessType(table,i);
-        }
-        // make it clickable to sort
-        headrow[i].sorttable_columnindex = i;
-        headrow[i].sorttable_tbody = table.tBodies[0];
-        dean_addEvent(headrow[i],"click", function(e) {
+	      if (mtch && typeof sorttable["sort_"+override] == 'function') {
```

Comment:
This adds tab-based indentation instead of spaces; later extra space indents are used where I think the indentation was correct. You might review and/or back out whatever superfluous changes in whitespace that you can.